README.md

```diff
@@ -105,21 +88,22 @@ Here are the steps to submit a proposal against one of the listed RFPs!
 1. Select an RFP you would like to apply for.
 2. Submit a PR against the [`rfp-proposal-template.md` file](https://github.com/filecoin-project/devgrants/blob/master/rfp-proposals/rfp-proposal-template.md) in the [`rfp-proposals` folder](https://github.com/filecoin-project/devgrants/tree/master/rfp-proposals) of this repo. Make sure to fill in all the categories of the RFP proposal template (incomplete proposals will not be considered). Rename your file `project-title.md`, making sure to replace `project-title` with the name of your project.
 
-**Remember to submit your PR before the wave deadline. PRs submitted after the deadline will be considered in the next wave (at which point, the RFP might be closed).**
-
 ### Submit a proposal for open grants
 
 Here are the steps to submit a proposal for the open grant category!
 1. Submit a PR against the [`open-proposal-template.md` file](https://github.com/filecoin-project/devgrants/blob/master/open-grant-proposals/open-proposal-template.md) in the [`open-grant-proposals` folder](https://github.com/filecoin-project/devgrants/tree/master/open-grant-proposals) of this repo. Make sure to fill in all the categories of the proposal template (incomplete proposals will not be considered). Rename your file `project-title.md`, making sure to replace `project-title` with the name of your project.
 
-
-**Remember to submit your PR before the wave deadline. PRs submitted after the deadline will be considered in the next wave (which means a 3-month delay in review for your proposal).**
-
+### Submit a proposal for microgrants
+                                                                                                                        
+Here are the steps to submit a proposal for a microgrant!
+1. Submit an issue on this repo. Make sure to fill in all the categories of the proposal [template](https://github.com/filecoin-project/devgrants/blob/master/microgrants/microgrants.md) (incomplete proposals will not be considered). Rename your file `[MICROGRANT]project-title.md`, making sure to replace `project-title` with the name of your project. 
```

Comment:
Use this link, which should automatically populate the various fields and templates. Try to simplify the instructions too (don't need the sentence on renaming the markdown file, since there's no .md file for this process).

https://github.com/filecoin-project/devgrants/issues/new?assignees=mishmosh&labels=type%3Amicrogrant&template=microgrant.md&title=Next+Step+Microgrant%3A+%3CYour+Title+Here%3E